_tour/tuples.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: tour
+title: Tuples
+
+discourse: true
+
+partof: scala-tour
+
+num: 
+next-page: mixin-class-composition
+previous-page: traits
+topics: tuples
+
+redirect_from: "/tutorials/tour/tuples.html"
+---
+
+In Scala, a tuple is a class that can hold elements of different types.
+Tuples are immutable.
+
+Tuples come handy when we have to return multiple values from a function.
+
+A tuple can be created as:
+
+```tut
+val ingredient = ("Sugar" , 25)
+
+println(ingredient.getClass) // class scala.Tuple2 
+```
+This creates a tuple containing a String element and an Int element.
+
+Tuple in Scala is a series of classes: Tuple2, Tuple3, etc., through Tuple22.
+So when we create a tuple with n elements(n lying between 2 and 22), Scala basically instantiates
+one of the corresponding class from the group, parametrized with types of constituent elements.
+For eg., ingredient is of type Tuple2[String, int].
+
+## Accessing the elements
+
+Tuple elements are accessed using underscore syntax.
+tuple._n gives nth element(given there are that many elements).
+
+```tut
+println(ingredient._1) // Sugar
+
+println(ingredient._2) // 25
+```
+
+## Iterating a tuple
+
+Tuple provides productIterator method to iterate it.
```

Comment:
I would suggest omitting any mention of `productIterator` (and related methods such as `productElement`) from the tour entirely. They are rarely used, since they don't provide any type-safety.